modules/402-ingress-nginx/images/controller-0-46/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN apt-get update \
   && luarocks install lua-iconv 7-3
 
 # IngressNginxController docker image
-FROM k8s.gcr.io/ingress-nginx/controller:v0.46.0@sha256:52f0058bed0a17ab0fb35628ba97e8d52b5d32299fbc03cc0f6c7b9ff036b61a as controller_0_46_0
+FROM registry.k8s.io/ingress-nginx/controller:v0.46.0@sha256:52f0058bed0a17ab0fb35628ba97e8d52b5d32299fbc03cc0f6c7b9ff036b61a as controller_0_46_0
```

Comment:
All nginx ingress pods of controllers >=0.46 will be restarted. Sounds like an impact to me.

Reply:
Adddd more changelog entries. Please, take a look